updating/understanding-the-update-service.adoc

```diff
@@ -14,4 +14,6 @@ If you are on a restricted network where disconnected clusters cannot access the
 
 include::modules/update-service-overview.adoc[leveloffset=+1]
 
+For more information about machine health checks, see xref:../machine_management/deploying-machine-health-checks.adoc#deploying-machine-health-checks[Deploying machine health checks].
```

Comment:
I suggest removing this link. I don't think that a user is going to find it at the right time. If your goals with this link is to get the user to find the new steps about pausing the MHC, I don't think that this is going to help.

Instead, I suggest considering moving the new procedure module to the upgrade docs themselves because that's when users are going to need to follow the steps. (@sagidlow, will you PTAL? I know you're doing some upgrade QOL improvements.)

Reply:
@kalexand-rh Thank you for your suggestions. I have addressed most of them. I consulted with @sagidlow about your suggestion and we discussed and agreed on the latest existing changes. Thank you, @sagidlow!